src/scss/base/_typography.scss

```diff
@@ -67,18 +67,26 @@ h6 {
 // For example: h3.h1 or h3.alpha
 h1,
 .alpha {
-    @include font-size(jumbo);
+    @include font-size(jumbo, true, narrow);
```

Comment:
Question; what are the last two parameters?

Reply:
(font-key, line-height output (boolean), subkey)

So here it’s saying, get me the font-size of jumbo for the narrow subkey, and output the line-height as well.